sydent/http/servlets/store_invite_servlet.py

```diff
@@ -125,6 +125,11 @@ def render_POST(self, request: Request) -> JsonDict:
         for k in extra_substitutions:
             substitutions.setdefault(k, "")
 
+        # MSC3288
+        substitutions["room_type"] = substitutions.pop(
+            "org.matrix.msc3288.room_type", ""
+        )
```

Comment:
I'm guessing these lines will go away when we move out of unstable

Reply:
Something like that. We'll probably have Synapse send both for a while and update Sydent to use just `room_type`? Or fallback to `org.matrix.msc3288.room_type`. 🤷